### PR TITLE
Analytics bugfix

### PR DESCRIPTION
**Pull request by soamvasani**
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1195)

### CODE REVIEW

File: router/analytics.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func (a *Analytics) run() {
 	ticker := time.NewTicker(24 * time.Hour)
 	for range ticker.C {
 		msg := a.gatherData()
-		msg.id = a.id
+		msg.ID = a.id
```
Comment (life1347) [inline router/analytics.go]: You may also want to handle err at L59.

Comment (soamvasani) [reply router/analytics.go]: I don't understand? The http post error is already handled?

Comment (life1347) [reply router/analytics.go]:
```
resp, err := http.Post(a.url, "application/json", bytes.NewReader(msgbytes))
```
No, right now we only handle the resp not err. If the err is not so critical, maybe we can ignore it?
```
resp, _ := xxxxx
```

Comment (soamvasani) [reply router/analytics.go]: Oh I see, good point and thanks for fixing it.

**Pull request merged**